### ISSUE TITLE
Quad lost size params

### ISSUE THREAD

**Issue by MyMyGames** [mindjolt/starling-builder-editor#8]
"Quad" lost params width & height  (version 0.6.2) in 0.6.1 everything worked

**Comment by ghost** [comment]
Sorry, I broke this one while fixing another bug.
I fixed it right away 632de92da79419aeda35deb2f2429afcd5a45994.

Before the next version pushes, please go to workspace -> edit template override and add Quad's params scaleX and scaleY as below, and reload the editor

```
        "cls":"starling.display.Quad",
        "params":[
            {"name":"width", "component":"slider", "min":1, "max":2000, "step":1, "read_only":true},
            {"name":"height", "component":"slider", "min":1, "max":2000, "step":1, "read_only":true},
            {"name":"scaleX", "step":0.01, "default_value":1},
            {"name":"scaleY", "step":0.01, "default_value":1},
            {"name":"color", "component":"colorPicker", "default_value":16777215}
        ],
```

Thanks for the report!

**Comment by ghost** [comment]
Fixed on v0.6.5.
